### ISSUE TITLE
Signing assertion

### ISSUE THREAD

**Issue by mrx23dot** [libbtc/libbtc#203]
Got this in production:
`
tx.c:880: btc_tx_sign_input: Assertion `sigderlen <= 74 && sigderlen >= 70' failed`

looks like assertion not always holds:
`assert(sigderlen <= 74 && sigderlen >= 70);`

I don't have testcase for it, but needs rethinking.

**Comment by zzzzzzch** [comment]
yeah. When the signature is serialized into DER format, the leading zeros of `R`/ `S` will be removed, which will cause the total length to be less than 70. I feel like the assertion here should be removed.
